### ISSUE TITLE
feat: remote state files as command-line option -f

### ISSUE THREAD

**Issue by Nuru** [roboll/helmfile#763]
In #648 you added the ability to specify remote state files in a state file using the syntax
```
helmfiles:
- # Terraform-module-like URL for importing a remote directory and use a file in it as a nested-state file
  # The nested-state file is locally checked-out along with the remote directory containing it.
  # Therefore all the local paths in the file are resolved relative to the file
  path: git::https://github.com/cloudposse/helmfiles.git@releases/kiam.yaml?ref=0.40.0
```

I expected that included with that feature would be the ability to use remote helmfiles on the command line, but it does not work:
```
$ helmfile -f 'git::https://github.com/cloudposse/helmfiles.git@releases/kiam.yaml?ref=0.40.0'
specified state file git::https://github.com/cloudposse/helmfiles.git@releases/kiam.yaml?ref=0.40.0 is not found
```

Please include this ability.

**Comment by mumoshu** [comment]
Hey! It has already been implemented in #670. And it works with the same URL you've provided on my machine:

```
helmfile -f 'git::https://github.com/cloudposse/helmfiles.git@releases/kiam.yaml?ref=0.40.0' diff
```

Are you by any chance running an older helmfile?

**Comment by Nuru** [comment]
Yes, sorry, I was using `helmfile` 0.73.1 and it seems you implemented this in 0.74.0. Closing this as already fixed.
